content/project/visual-changelogs/visualchangelog342/index.md

```diff
@@ -406,6 +406,18 @@ Users can define a custom tolerance by adding a `custom tolerance` property to t
 
 This feature was developed by [Jean Felder](https://github.com/ptitjano)
 
+## Browser 
+
+### Feature: Postgreprovider hide raster overviews
```

Comment:
PostgreSQL provider

Reply:
moved down to the other PR in data providers